lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -1143,7 +1143,7 @@ namespace Js
         if (nestedCount > 0)
         {
             nestedArray = RecyclerNewPlusZ(m_scriptContext->GetRecycler(),
-                nestedCount*sizeof(FunctionProxy), NestedArray, nestedCount);
+                (nestedCount-1)*sizeof(FunctionProxy*), NestedArray, nestedCount);
```

Comment:
looks c++ 11 does not allow it, but the compilers do allow zero length array in structure defination. updated the PR